aviary/subsystems/propulsion/utils.py

```diff
@@ -252,10 +252,28 @@ def setup(self):
 
 
 # class InstallationDragFlag(Enum):
-#     '''
+#     """
 #     Define constants that map to supported options for scaling of installation drag.
-#     '''
+#     """
 #     OFF = auto()
 #     DELTA_MAX_NOZZLE_AREA = auto()
 #     MAX_NOZZLE_AREA = auto()
 #     REF_NOZZLE_EXIT_AREA = auto()
+
+
+class PropModelVariables(Enum):
```

Comment:
Prop -> Propeller

Otherwise it's easy to confuse with propulsion

Reply:
done.